### ISSUE TITLE
Add Support for Instantiated Widgets in WP 4.6

### ISSUE THREAD

**Issue by PodsBot** [pods-framework/pods#3667]
Add Support for Instantiated Widgets in WP 4.6 submitted via Slack by jimtrue

**Comment by jimtrue** [comment]
This is in reference to this Trac ticket #35990 https://core.trac.wordpress.org/ticket/35990

This could be added to Supports under Advanced Options allowing existing WP posts or taxonomy widgets to be activated automatically, I think.

**Comment by JoryHogeveen** [comment]
If this gets added in core, Pods doesn't have to do anything right?

**Comment by jimtrue** [comment]
@JoryHogeveen this only adds support for Instatiated widgets. It doesn't activate them automatically. That's what this would do: just add the ability to 'turn on Recent Posts, etc.' widgets in the Advanced Options. A 'nice to have' that folks who aren't comfortable with PHP could do.

**Comment by jimtrue** [comment]
Or at least that's how I read the Trac Ticket.

**Comment by JoryHogeveen** [comment]
The attached diff implies that it does: https://core.trac.wordpress.org/attachment/ticket/35990/35990.0.diff
It simply loops through all public post_types and creates a widget for these.

The last comment on this ticket asked this to be added in the register function instead so the user/developer can choose wether to give support or not. I'd like to see that implemented aswell.

Guess we'll need to wait on the actual implementation.

**Comment by jimtrue** [comment]
Yep, wait and see if it's in the 'register' so it's up to the individual developer or not; if it's in register, we only need to add it to the 'supports' section in Advanced Options, I think.
